en/ui-system/components/editor/webview.md

```diff
@@ -207,4 +206,4 @@ Because of the limitations of Web platform, it can not be implemented by this me
 </html>
 ```
 
-#### Note: Stressed once more Cross-domain issues on Web platform need to be resolved by you, the developer.
+> **Note**: to stress once more: cross-domain issues on Web platform need to be resolved by you, the developer.
```

Comment:
**Again: cross-domain issues on Web platform need to be resolved by yourself.**

Reply:
We don't actually talk like this.

What I wrote is correct, or we could also say:

cross-domain issues on Web platform need to be resolved by you, yourself.

or

Again: cross-domain issues on Web platform need to be resolved by yourself as __Cocos Creator__ cannot assist with this.